### PR TITLE
Image overlayer 1.0.5

### PR DESCRIPTION
**Pull request by imranMnts**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by imranMnts** [comment]
@shams-ahmed can you please have a look
small improvements compared to 1.0.2 which are required in our project

File: steps/image-overlayer/1.0.5/step.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+title: Image overlayer
```
Comment (godrei) [inline steps/image-overlayer/1.0.5/step.yml:1]: Based on the [Step Title conventions](https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title) `Add image overlay` (or something similar) would be a better title.

**Comment by godrei** [comment]
Hi @imranMnts ,
I left one note for the Step's title, would be great to update it in future releases.
Other than that it looks good to me, thanks for the new version.

**Review by godrei — APPROVED** [review]

File: steps/image-overlayer/1.0.5/step.yml (hunk 1 above)
Comment (imranMnts) [reply steps/image-overlayer/1.0.5/step.yml:1]: I took the point, thank you 🙂

**Pull request merged**